src/main/java/com/demo/classroom/Service/AuthService.java

```diff
@@ -30,13 +39,23 @@ public class AuthService {
 
     private PasswordEncoder passwordEncoder;
 
+    private AuthenticationManager authenticationManager;
+
+    @Autowired
+    private JwtService jwtService;
     @Autowired
-    public AuthService(UserRepository userRepository, TeacherRepository teacherRepository, StudentRepository studentRepository, PasswordEncoder passwordEncoder){
+    private UserService userService;
```

Comment:
It is better to use constructor injection consistently, as you're already doing for the repositories and other services. This is safer, especially for testing, and aligns with best practices.

Reply:
Done